ts/routes/deck-options/FsrsOptions.svelte

```diff
@@ -148,7 +148,13 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
                         ignoreRevlogsBeforeMs: getIgnoreRevlogsBeforeMs(),
                         currentParams: params,
                     });
-                    if (
+
+                    if (!resp.fsrsItems) {
+                        setTimeout(
+                            () => alert(tr.deckConfigFsrsParamsNoReviews()),
+                            200,
+                        );
```

Comment:
```suggestion
                        );
                        $config.fsrsParams5 = defaults.fsrsParams5
```
I'm wondering if it should also clear any parameters that are already in the box if the user has 0 reviews on that preset.

Reply:
Not sure on this one. But, why should Anki do something which is not required?

Reply:
I'm not convinced on this myself, If a user optimises a preset with no reviews but still has parameters because they cloned the preset for example I feel like "optimise" should set it to the optimal, which in the case of an empty preset would be the default.

But yeah I think you're right.